### DISCUSSION TITLE
assertJsonContains test assertion

### DISCUSSION THREAD

**Discussion by binaryfire** [laravel/framework#46686] [Ideas]
It'd be great to have `assertJsonContains` to use in cases like this:

```
  Unable to find JSON:

[{
    "error": "The file field must be a file of type"
}]

within response JSON:

[{
    "message": "The file field must be a file of type: image/jpeg, image/png.",
    "errors": {
        "file": [
            "The file field must be a file of type: image/jpeg, image/png."
        ]
    }
}].
```

There are a lot of situations where part of the returned JSON may change depending on the app's current configuration.

**Comment by JurianArie** [comment]
Changing a key is a breaking change for apis. So I would expect it to fail. If your configuration changes behavior, you should be testing the different outcomes anyways.

You might also want to look at https://laravel.com/docs/10.x/http-tests#assert-invalid
